test/inductor/test_torchinductor_opinfo.py

```diff
@@ -1126,7 +1125,7 @@ def tearDown(self):
     @skipCUDAMemoryLeakCheckIf(
         True
     )  # inductor kernels failing this test intermittently
-    @skipCUDAIf(not HAS_CUDA_AND_TRITON, "Skipped! Triton not found")
+    @requires_cuda_and_triton
```

Comment:
This replacement is blocking other device, eg, cpu. xpu.

Reply:
@janeyx99 @ghostspiders  should we revert this PR?

Reply:
Oh no, what is going wrong? requires_cuda_and_triton should do the same thing as the previous skipIf 

ah i see you have a PR, I'll continue the discussion there